[Ancient Station] First Round Of Changes/Additions

Steelpoint opened the pull request:
First main changes based on feedback and similar.

* Fixed minor mapping issues, such as broken glass and similar.
* Added some bluespace crystals to the Hivebot ship for RnD.
* All station helmets and the hardsuit helmet no longer block vision.
* NASA Engineering Void Suit now has a proper slowdown of 4.
* Two pairs of 'worn down insulated gloves' have been added to engineering. Think of them as Budget Gloves, but they have a much better chance of working.
* New movement sounds for the prototype RIG Hardsuit, much more fitting.
* Minor map changes, such as a extra oxygen cannister and similar.

Fixes https://github.com/tgstation/tgstation/issues/28947

:cl: Steelpoint (Ancient Station)
add: Major changes to Ancient Station include new sounds for the prototype RIG hardsuit, the NASA Engineering Voidsuit slowing the user down properly and new insulated gloves in engineering.
add: Several minor changes, such as an extra oxygen tank, spawn in equipment and minor tile changes.
/:cl:

Steelpoint committed:
InitialCommit

DoctoreMad commented:
>Slowdown of 4

Isn't that unnecessarily painful.

Kraseo commented:
there's no slowdown in space

Steelpoint commented:
Its to discourage them to wear it indoors, in space there's no movement penalty.

Ergovisavi commented:
Consider that any slowdown on non armored gear is entirely irrelevant because they will just hold it in their hands, and that having the slowdown is just annoying for everyone involved.

I get the intent, but in practice all you're doing is being annoying.

DoctoreMad commented:
Can you give botany some roundstart saltpetre(maybe) and pest spray(DEFINITELY)? Grapes contain sugar which increase pest levels, but sometimes its the only plant that is gaining potency.

DoctoreMad commented:
Oh
I see

PraiseRatvar commented:
> All station helmets and the hardsuit helmet no longer block vision.

not a fan of this nerf, like people said on your feedback thread, git gud

Steelpoint commented:
I'll see how the general mood before I commit that change, I've still got other things to work on for this PR.

Steelpoint committed:
BannersAndFixes

Steelpoint commented:
I've removed the NT Banners as I did not realise they can be used to heal people, I've also made some minor spelling changes.

DoctoreMad commented:
Very rude

Repukan commented:
Don't nerf the helmets, they're cool like that.

Also the banners are hilarious.

Steelpoint commented:
I'll likely keep the helmet visor limitation.

Cyberboss commented on code/modules/clothing/gloves/color.dm:
```diff
@@ -22,6 +22,21 @@
+
+
+/obj/item/clothing/gloves/color/fyellow/New()
+	..()
```
Initialize and . = ..()

Steelpoint replied:
Don't know what your asking me to change as its identical to the other similar code entry for budget gloves.

juanototo2 commented:
The banners are ridiculous but they are hilariously fun. Keep the "whacking people with toolboxes while waving around a flag that instantly closes wounds" meme. PLEASE.

Jalleo commented:
Perhaps make it so the old helmets visors are toggleable? When up the helmet has half protection or something like that.

PraiseRatvar commented:
also please add an airlock somewhere on delta or charlie so people don't have to make a giant gaping hole for mechs or borgs

Armhulen commented:
WHY DO PEOPLE ENJOY WELDER VISION JESUS

Repukan commented:
We don't, but if we add QoL changes then it follows more QoL will be added, thus killing the spirit of the ruin.

Slibbery slobe

MimicFaux commented:
The entire point of the ruin is to enjoy a hard-mode reclamation project, and so far 9/10 runs on it have been made fun by the difficulty level and roleplay (probably a bug) with other survivors. Everything from growing plants for healing, multiple trips to recharge shitty guns, to electrocuting myself on the solar arrays because someone else turned the grid on, have made the experience for me.

I like the vision reduction, I enjoyed the lack of gloves (but won't complain for budget gloves+), there was even something thematic about picking through the chemical storage beakers. What I would like to see is some more piece to the remains of atmos added for additional projects the spessmen can work on.

Steelpoint commented:
PR is still a work in progress, I'll post again when my next set of changes and alterations is up.

Really-Good-Soda-Flavor commented:
These look like stupid, unnecessary changes that nobody asked for to me.

ShizCalev commented on _maps/RandomRuins/SpaceRuins/oldstation.dmm:
```diff
@@ -5204,6 +5208,46 @@
+/obj/item/weapon/stock_parts/console_screen,
+/turf/open/floor/plasteel/white/side{
+	tag = "icon-whitehall (EAST)";
+	icon_state = "whitehall";
```
dirty vars
(you can just remove these two lines with notepad or w/e tbh)

Steelpoint committed:
NewChanges

Steelpoint commented:
New push has some changes. Helmets and visors have the tinting again, I've removed the airless tiles on Charlie Station since I think that just raises unnecessary issues with broken atmo, and the fact its a non-issue to resolve. I've also scattered the parts to construct a sleeper, if you can find the parts.

Also some other minor mapping changes.

Repukan commented:
Don't add parts to build a sleeper, they're already available if you deconstruct other machines.

Armhulen commented:
at least not the hardsuit having tint aw god compromise

Tacolizard commented:
the voltage in the solars made it so that cutting or laying down any wire connected to them would shock you. It is better that now we get some insulation, but it's dumb to basically force the player to shock themselves in order to wire the solars.

Tacolizard commented:
also this map shouldn't be called oldstation (like it is in the filename) because that's the name of the original pre-boxstation map.

PraiseRatvar commented:
just it call it steelstation

Steelpoint commented:
I should mention I have removed the floor tiles on the solar array.

Ergovisavi commented:
> I've removed the NT Banners as I did not realise they can be used to heal people, I've also made some minor spelling changes.

Keep them in but change the effect to only work if you're below 20 HP and not in crit or something. It boosts morale, but that'll only keep you trucking along, it won't completely heal you.

You keep some of the absurdity for fun, but keep it from turning you into fucking wolverine.

DoctoreMad commented:
Why do the transit tubes have huge kPa anyways?

Steelpoint commented:
I got some further changes I'm making.

I might add blueprints to help in construction.

Steelpoint committed:
FinalChanges

Steelpoint commented:
Pushed some changes, so my PR is now merge ready.

* The three spawn roles now have some extra equipment on them to differentiate them a bit. Security Officer spawns with a flash and handcuffs (plus his uniform having 10 melee makes him more combat focused). Engineer spawns with a emergency oxygen tank and a worn our insulated glove. Scientist spawns with a bandage in their pocket.
* Removed the tinting on the hardsuit helmet, I think that's fair considering its slow down.
* Reworked minor tiles, such as the solar arrays. Also got rid of the excess items to reconstruct a sleeper without raiding other machines.

Steelpoint commented:
Also @DoctoreMad the transit tubes had high pressure due to the lack of air on Charlie Station seeping into the tubes.

My removal of the airless tiles on Charlie should stop this issue.

AnturK commented on code/modules/clothing/gloves/color.dm:
```diff
@@ -22,6 +22,20 @@
+	item_color="yellow"
+	resistance_flags = 0
+
+/obj/item/clothing/gloves/color/old/New()
```
Initialize, also this should just be subtype of fake yellows.

Steelpoint replied:
What does initialize mean?

AnturK replied:
It's a function we call on objects immediately once map is fully loaded and ready. Replace New with Initialize, we have more control over initialization times and process with it. See https://github.com/tgstation/tgstation/blob/master/code/controllers/subsystem/atoms.dm if you want more details.

Steelpoint committed:
InitializeFix

Steelpoint commented:
Asked for changes have been made, PR is merge ready assuming nothing breaks.

Steelpoint committed:
Typo

lordpidey commented:
So... how does an area with LOW pressure seeping into the tube cause HIGH pressure?  That seems like it's a bug in it's own right.

Steelpoint commented:
This is still good to merge. I addressed the requested changes as @Cyberboss requested.

octareenroon91 commented:
How many oxygen tanks are there overall?

Steelpoint commented:
Two I believe.

Steelpoint commented:
So, err, is this going to get merged or anything anytime soon?

@Cyberboss for approving the changes I made.

Cyberboss commented on code/modules/clothing/gloves/color.dm:
```diff
@@ -22,6 +22,14 @@
+	name = "worn out insulated gloves"
+
+/obj/item/clothing/gloves/color/fyellow.old/Initialize()
+	..()
```
. = ..()

Cyberboss commented on code/modules/clothing/gloves/color.dm:
```diff
@@ -22,6 +22,14 @@
+	desc = "Old and worn out insulated gloves, hopefully they still work."
+	name = "worn out insulated gloves"
+
+/obj/item/clothing/gloves/color/fyellow.old/Initialize()
```
/

Steelpoint committed:
MinorFixes

Steelpoint commented:
Fixes have been deployed.

Steelpoint committed:
MoreFixes
